### ISSUE TITLE
add rpc to change commission fee

### ISSUE THREAD

**Issue by liaoyd** [pchain-org/pchain#50]
Candidate have to cancel candidate and apply candidate again if he wanna change his commission fee. While after cancel candidate, all proxied amount from delegator will be refunded, which is very inconvenient.

**Comment by bertux** [comment]
I fully agree, I hope this bug will be corrected soon but I don't expect much because not many bugs have been corrected already :-(
